func err_func(int64 a) -> int64[3] a
{ }
